sktalk/corpus/utterance.py

```diff
@@ -12,34 +14,31 @@ class Utterance:
     participant: Optional[str] = None
     time: Optional[list] = None
     begin: Optional[str] = None
+    begin_timestamp: Optional[str] = None
     end: Optional[str] = None
-    metadata: Optional[dict[str, Any]] = None
-    utterance_clean: Optional[str] = None
+    end_timestamp: Optional[str] = None
+    utterance_raw: Optional[str] = None
     utterance_list: Optional[list[str]] = None
     n_words: Optional[int] = None
     n_characters: Optional[int] = None
-    time_to_next: Optional[int] = None
-    dyadic: Optional[bool] = None
     FTO: Optional[int] = None
+    metadata: Optional[dict[str, Any]] = None
 
     def __post_init__(self):
-        # clean utterance:
-        if not self.utterance_clean:
-            self._clean_utterance()
-
-        # generate a list of words in the utterance
-        if not self.utterance_list:
-            self.utterance_list = self.utterance_clean.split()
+        if not self.utterance_raw:  # if reading in existing data, we do not want to overwrite the raw utterance
```

Comment:
Should there be a clear distinction between an empty string and `None`? I am not sure an empty string should be overwritten.
```suggestion
        if self.utterance_raw is None:  # if reading in existing data, we do not want to overwrite the raw utterance
```